qcodes/dataset/descriptions/versioning/converters.py

```diff
@@ -0,0 +1,111 @@
+from typing import Dict, List
+
+from qcodes.dataset.descriptions.param_spec import ParamSpec, ParamSpecBase
+from qcodes.dataset.descriptions.dependencies import InterDependencies_
+from qcodes.dataset.descriptions.versioning.v0 import InterDependencies
+import qcodes.dataset.descriptions.versioning.v0 as v0
+import qcodes.dataset.descriptions.rundescriber as current
+
+
+def old_to_new(idps: InterDependencies) -> InterDependencies_:
```

Comment:
well, as a person who worked with this thing for a bit, i got used to this name already. it's not the best. but perhaps we can keep it in order not to spend the effort into renaming? the "new" interdependencies will soon go away because of the inception of paramspectrees. @WilliamHPNielsen what do you think?

Reply:
Yes, but I was thinking we could reserve that for the future.